backend/src/main/resources/application.yml

```diff
@@ -38,3 +38,6 @@ management.endpoint.health.show-details: always
 management.health:
   db.enabled: true
   diskspace.enabled: false
+scheduler.cron:
```

Comment:
Optional: shall we use double-quotes `"` instead of single ones as in application.yaml everywhere we have `"`

Reply:
I have read that the cron definition should be in the single quotes there.
i.e. https://medium.com/@ksivams/scheduled-job-cron-variable-into-configuration-file-9e46b47b0de7

Reply:
Tak zajimavy, tak to asi nevadi.